### PR TITLE
Fix newline parsing in MD to HTML parser

### PR DESCRIPTION
**Pull request by parasharrajat**
@Jag96  will you please review this?

[Explanation of the change or anything fishy that is going on]
If a `br` tag is followed by a newline then we should parse both as single newline. e.g.
` <b>Joe likes to eat <b><br>\n Apple & Doughnut.`

### Fixed Issues
$ https://github.com/Expensify/Expensify.cash/pull/4009#issuecomment-879534328

# Tests
1. What unit/integration tests cover your change? What autoQA tests cover your change?
UNit
1. What tests did you perform that validates your changed worked?
Unit

# QA
1. What does QA need to do to validate your changes?
NA
1. What areas to they need to test for regressions?
NA

### CODE REVIEW

File: lib/ExpensiMark.js [hunk 1]
```diff
@@ -174,7 +174,7 @@ export default class ExpensiMark {
 
                 // Replaces open and closing <br><br/> tags with a single <br/>
                 pre: inputString => inputString.replace('<br></br>', '<br/>').replace('<br><br/>', '<br/>'),
-                regex: /<br\s*[/]?>/gi,
+                regex: /<br\s*[/]?>\n?/gi,
```
Comment (Jag96) [inline lib/ExpensiMark.js:177]: Can you update one of the tests to ensure this case is covered?

Comment (parasharrajat) [reply lib/ExpensiMark.js:177]: Ok

**Comment by parasharrajat** [comment]
Done

**Review by Jag96 — APPROVED** [review]
LGTM!

**Pull request merged**